3rdparty/ippicv/ippicv.cmake

```diff
@@ -38,6 +38,11 @@ function(download_ippicv root_var)
     return()
   endif()
 
+  if(HUNTER_ENABLED)
```

Comment:
Do we care to wrap the other `hunter_add_package` calls in `if(HUNTER_ENABLED)`? It's not strictly necessary in the fork as the fork is meant to be used for hunter anyways

Reply:
I am a noob but I guess the main advantage of that would be if these patches were merged to upstream, they would work for both HUNTER_ENABLED and !HUNTER_ENABLED cases